src/main/java/it/pagopa/pn/deliverypush/action/utils/TimelineUtils.java

```diff
@@ -276,6 +276,7 @@ public TimelineElementInternal buildSendSimpleRegisteredLetterTimelineElement(In
                 .productType(productType)
                 .numberOfPages(sendResponse.getNumberOfPages())
                 .envelopeWeight(sendResponse.getEnvelopeWeight())
+                .requestId(requestId)
```

Comment:
usare prepareRequestId, in modo da rendere più chiaro che cosa si sta salvando

Reply:
resolved with commit ac0d3f71ccdd88155587af92c33c47243ad239e1